src/agni/client/view/LoginView.java

```diff
@@ -1,28 +1,20 @@
 package agni.client.view;
 
-import java.awt.event.FocusListener;
-import java.awt.event.MouseEvent;
-import java.awt.event.MouseListener;
-
 import agni.client.action.*;
 import agni.server.sender.StatusSender.Status;
 import charva.awt.BorderLayout;
-import charva.awt.Color;
 import charva.awt.Container;
 import charva.awt.event.ActionEvent;
 import charva.awt.event.ActionListener;
 import charva.awt.event.KeyEvent;
 import charva.awt.event.KeyListener;
-import charvax.swing.BoxLayout;
 import charvax.swing.JFrame;
 import charvax.swing.JLabel;
 import charvax.swing.JMenu;
 import charvax.swing.JMenuBar;
 import charvax.swing.JMenuItem;
-import charvax.swing.JPanel;
 import charvax.swing.JTextArea;
 import charvax.swing.JTextField;
-import charvax.swing.SwingConstants;
```

Comment:
Probably doesn't belong in this PR but it's pretty harmless and it is useful, so I think it's a good idea to keep :+1: